src/SFA.DAS.ReferenceData.Application/SFA.DAS.ReferenceData.Application.csproj

```diff
@@ -63,6 +63,9 @@
     <Compile Include="Queries\GetEducationalOrganisations\FindEducationalOrganisationsHandler.cs" />
     <Compile Include="Queries\GetEducationalOrganisations\FindEducationalOrganisationsQuery.cs" />
     <Compile Include="Queries\GetEducationalOrganisations\FindEducationalOrganisationsResponse.cs" />
+    <Compile Include="Queries\GetIdentifiableOrganisationTypes\GetLocateableOrganisationTypesHandler.cs" />
```

Comment:
Can the query also be renamed to match the action?

Reply:
Ahhh - changed the class names, forgot the files. Done.